.github/workflows/module-prep-release.yml

```diff
@@ -0,0 +1,128 @@
+name: Prepare Release
+# This workflow is manually dispatched. It:
+# - bumps the version as specified in the input
+# - prepares build artifacts
+# - creates a release branch
+# - commits the changes
+# - creates a pull request to the main branch
```

Comment:
Can we move this doc block above the job below? 

I've found it really helpful to have:
- a one sentence description of the workflow before the name.
- A block before each job lists the steps that taken in that job.
- If necessary, a fine-grained comment above a step explaining why something is a certain way.

[Here is a basic example in WP Core](https://github.com/WordPress/wordpress-develop/blob/trunk/.github/workflows/reusable-build-package.yml).

Reply:
Updated as a doc block in the job.